src/main.js

```diff
@@ -18,6 +19,12 @@ function createWindow() {
 }
 
 app.on('ready', () => {
+  installExtension.default(installExtension.REACT_DEVELOPER_TOOLS)
+    .then((name) => console.log(`Added Extension:  ${name}`))
+    .then(() => installExtension.default(installExtension.REDUX_DEVTOOLS)
+    .then((name) => console.log(`Added Extension:  ${name}`)))
+    .catch((err) => console.log('An error occurred: ', err))
```

Comment:
I'm going back and forth on this. I like the consistency of every function using params. But sometimes it is nice to leave them out. 

Let's pick a side and make our linter yell at us if we stray!